lib/titiler-pgstac-api/runtime/src/handler.py

```diff
@@ -22,6 +20,9 @@
     }
 )
 
+from titiler.pgstac.main import app  # noqa: E402
```

Comment:
Here I am fixing a bug introduced in https://github.com/developmentseed/eoapi-cdk/pull/61.

When you import from `titiler.pgstac.main`, you trigger the creation of the postgres settings https://github.com/stac-utils/titiler-pgstac/blob/50e37883ffe12cc739467f25d370c00672fe9070/titiler/pgstac/main.py#L42. So, before doing that import, you need to make sure all the variables you need for these settings are in the environment. 

I swapped the import and the postgres env setup, and turned off the flake8 E402 rule ("module level import not at top of file") for these imports.

Reply:
oh Sorry about this 🙏